src/post/workshed.ts

```diff
@@ -83,10 +83,11 @@ const worksheds = [
         print("Reconfiguring trainset, as our next station is empty", "blue");
         return TrainSet.setConfiguration(getBestCycle());
       } else {
-        const bestTwoStations = getBestCycle().splice(0, 2);
+        const offsetWanted = getBestOffset();
+        const stationToWatchFor = getBestCycle()[offsetWanted];
         const offset = get("trainsetPosition") % 8;
-        if (bestTwoStations.includes(TrainSet.next())) return;
-        print(`Reconfiguring trainset, as our next station is ${TrainSet.next()}`, "blue");
+        if (TrainSet.next() !== stationToWatchFor) return;
```

Comment:
Might be *slightly* non-optimal if we run turns post garbo (to grab the volcoino or do a melf dupe), and we skip by `stationToWatchFor` for the overdrunk/rollover leg (which we will then have to wait a full cycle before garbo reconfigures the trainset).